ultimatejobweb/models.py

```diff
@@ -8,9 +7,20 @@ class user(models.Model):
     profession = models.CharField(max_length=128)
```

Comment:
I wouldn't call the class user. Maybe UserProfile.
From our discussion yesterday, you decided that you didn't want this table/field.

Reply:
That's correct - also will be fixed today evening